requirements.txt

```diff
@@ -1,4 +1,7 @@
-ansible==2.10.7
+ansible==9.3.0
+ansible-base==2.10.17
+ansible-core==2.16.4
```

Comment:
Some roles are not ready for ansible-core > v2.14:

```
ERROR! [DEPRECATED]: ansible.builtin.include has been removed. Use include_tasks or import_tasks instead. This feature was removed from ansible-core in a release after 2023-05-16. Please update your playbooks.
```

While ansible-core v2.14 requires ansible v7, it seems that this setup works fine without needing ansible-base v2.10

```
ansible==7.7.0
ansible-core==2.14.14
```

Reply:
Ok, I was aware of that and wanted to do a Mmmmmmega PR! with all of the roles updated, but wanted to keep this one small. Quite happy to update all of the playbooks and roles, it won't change functionality. What do you think?

Reply:
Sounds good! How about we proceed with merging this PR, which bumps the ansible/ansible-core versions in a backwards compatible way and adds the lint workflow for the playbooks?

Then, you can submit a separate Mmmmmmega PR specifically for updating all the playbooks and roles for the latest & greatest ansible/ansible-core.

Reply:
I think a better approach might be to tag a release to the HEAD just before we merge this, and then declare a breaking change when we merge?

Reply:
Done in bc6cbeb